src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,28 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private float diggingDepth;
+
     public Excavator() {
     }
 
+    public Excavator(String name,String color,float diggingDepth) {
+        super();
+        this.setName(name);
+        this.setColor(color);
```

Comment:
same here 
it's better to add a constructor to the Machine class and inherit it here like that 
```suggestion
        super(name, color);
```